### PR TITLE
{lib}[GCCcore/9.3.0] libgeotiff v1.5.1

### PR DESCRIPTION
**Pull request by smoors**
(created using `eb --new-pr`)
depends on #10154

### CODE REVIEW

**Comment by smoors** [comment]
Test report by @smoors
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node367.hydra.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/3b147aaf73898212a5204088c023011c for a full test report.

**Comment by zao** [comment]
Test report by @zao
**SUCCESS**
Build succeeded for 2 out of 2 (1 easyconfigs in this PR)
freja - Linux ubuntu 18.04, Intel(R) Core(TM) i7-8700K CPU @ 3.70GHz, Python 2.7.17
See https://gist.github.com/275d87f9f7bf51122c49760cfde9f3b6 for a full test report.

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
skalman.hpc2n.umu.se - Linux ubuntu 18.04, Intel(R) Core(TM) i7-8565U CPU @ 1.80GHz, Python 2.7.17
See https://gist.github.com/0de29bf974605735cc35f5b48589a4d0 for a full test report.

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/a29ee00f6c3b10f518ba95773d5bf886 for a full test report.

File: easybuild/easyconfigs/l/libgeotiff/libgeotiff-1.5.1-GCCcore-9.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+sources = [SOURCE_TAR_GZ]
+checksums = ['f9e99733c170d11052f562bcd2c7cb4de53ed405f7acdde4f16195cd3ead612c']
+
+builddependencies = [
+    ('binutils', '2.34'),
+]
+
+dependencies = [
+    ('PROJ', '7.0.0'),
+    ('libjpeg-turbo', '2.0.4'),
```
Comment (akesandgren) [inline easybuild/easyconfigs/l/libgeotiff/libgeotiff-1.5.1-GCCcore-9.3.0.eb]: Please list LibTIFF and zlib here as they are direct dependencies and explicitly referenced below

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
login2.cerberus.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6126 CPU @ 2.60GHz, Python 2.7.5
See https://gist.github.com/1571079a86624292187d1864ac53d54f for a full test report.

**Comment by lexming** [comment]
my test is with last commit (I'm faster than you @smoors!)

File: easybuild/easyconfigs/l/libgeotiff/libgeotiff-1.5.1-GCCcore-9.3.0.eb (hunk 1 above)
Comment (smoors) [reply easybuild/easyconfigs/l/libgeotiff/libgeotiff-1.5.1-GCCcore-9.3.0.eb]: @akesandgren ok, done

**Comment by smoors** [comment]
Test report by @smoors
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node355.hydra.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/d78e0daf1d4a177c350dd9afb592fe45 for a full test report.

**Comment by lexming** [comment]
Going in, thanks @smoors !

**Pull request merged**